libs/confman/testing/tests/file_storage_test.cpp

```diff
@@ -134,5 +141,34 @@ auto main(int argc, char** argv) -> int {
     ut::expect(called == 1);
   };
 
+  "backup on config change"_test = [&] {
+    file_testable<test_me> conf{ ctx, "/tmp/test_uuid", test_me{ .a = observable<int>{ 3 }, .b = "bar" } };
+    conf.make_change()->a = 2;
+    ctx.run_for(std::chrono::milliseconds(1));
+    auto files = std::filesystem::directory_iterator("/tmp");
+
+    bool backup_found = false;
+    std::string found_file;
+
+    for (const auto& file : files) {
+      if (file.path().string().find("/tmp/test_uuid_") != std::string::npos) {
+        backup_found = true;
+        found_file = file.path().string();
+        break;
+      }
+    }
```

Comment:
please use https://en.cppreference.com/w/cpp/filesystem/exists

Reply:
Since the full path of the file is not known it's not possible to use `std::filesystem::exists`. A uuid is concatenated on the back of `/tmp/test_uuid_` which is random.